mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -6818,6 +6818,81 @@ msgstr "@*!^: 0H... @ n3wc0m3r! H0vv p1d y0v g3t he7e? <NONE> h3llen\"T $33N %^&
 "@*!^: WElcoME! W3lcome t0... 0H N/A: We never g0t ar0und to gi^ing 0ur town {<1} We never g00000000 {<2} W3 neVer Bu1lt 0uR t0wN\n"
 "TH S D ES N T EX ST Y U AR  C NNE  ED TO TH  V ID YO  M ST LEA E N W"
 
+msgid "buddha_welcome"
+msgstr "Welcome to Buddha Mountain!"
+
+msgid "old_sphalian_house01"
+msgstr "You watch some TV with your beloved Lv ${{monster_0_level}} ${{monster_0_name}}.,0"
+
+msgid "old_sphalian_house02"
+msgstr "Home Sweet Home"
+
+msgid "old_sphalian_center01"
+msgstr "Your tuxemon are now at full health!"
+
+msgid "old_sphalian_center02"
+msgstr "It says that it is under repair"
+
+msgid "old_sphalian_town01"
+msgstr "Sphalian Town - a quiet\, peaceful place"
+
+msgid "rubberduck_cave_01"
+msgstr "A good place to hide an item"
+
+msgid "rubberduck_cave_02"
+msgstr "if i could swim..."
```

Comment:
capital letter required at the start of the sentence here and in more places below.  i know that these issues were merged before we had strong requirements for this, and its not your problem, but it would be nice if we can fix problems when we touch code that has them.

Reply:
no problem